src/shell/lisa_shell

```diff
@@ -93,13 +93,27 @@ fi
 echo -ne "$LISASHELL_DEFAULT"
 }
 
+function _lisa-update-all {
+echo -ne "${LISASHELL_BLUE}"
+echo -e "*** Update LISA installation"
+
+git pull
+
+_lisa-update-submodules
+
+echo -ne "$LISASHELL_DEFAULT"
+}
+
 function lisa-update {
 CMD=${1:-submodules}
```

Comment:
it does if you've manually updated `origin/master`.  In any case, if you prefer "all" to be the default I'll change it.